corehq/apps/reports/standard/monitoring.py

```diff
@@ -586,6 +586,10 @@ def get_row(self, user=None):
         first_col = self.get_raw_user_link(user) if user else _("Total")
         return [first_col] + styled_date_cols + [sum(date_cols)]
 
+    def get_raw_user_link(self, user):
+        from corehq.apps.reports.standard.inspect import SubmitHistory
+        url = SubmitHistory.get_url(domain=self.domain)
+        return super(DailyFormStatsReport, self).get_raw_user_link(user, url)
```

Comment:
This does feel a little hacky. Conceptually you usually want the subclass's method to expand upon the superclass's method's signature, and pass through anything in their shared signature unchanged. Here, you added the link variable to the superclass's method as a way for a subclass to hijack it. (`link` isn't *really* intended to be part of `get_raw_user_link` in any of the calling code.) If what url to use is something you want to override, I'd suggest making it a separate method (`get_user_link_url`), calling it from `get_raw_user_link` (instead of calling `CaseListReport.get_url` directly) and overriding it in `DailyFormStatsReport`. Then it becomes very clear how you're supposed to use each part.